.github/workflows/scripts/generate-playground-blueprint.js

```diff
@@ -0,0 +1,78 @@
+const https = require( 'https' );
+
+const generateWordpressPlaygroundBlueprint = ( runId, prNumber ) => {
+	const defaultSchema = {
+		landingPage: '/wp-admin/admin.php?page=wc-admin',
+
+		preferredVersions: {
+			php: '8.0',
+			wp: 'latest',
+		},
+
+		phpExtensionBundles: [ 'kitchen-sink' ],
+
+		features: { networking: true },
+
+		steps: [
+			{
+				step: 'installPlugin',
+				pluginZipFile: {
+					resource: 'url',
+					url: `https://playground.wordpress.net/plugin-proxy.php?org=woocommerce&repo=woocommerce&workflow=Build%20Live%20Branch&artifact=plugins-${ runId }&pr=${ prNumber }`,
+				},
+				options: {
+					activate: true,
+				},
+			},
+			{
+				step: 'login',
+				username: 'admin',
+				password: 'password',
+			},
+		],
+		plugins: [],
+	};
+
+	return defaultSchema;
+};
+
+async function run( { github, context, core } ) {
+	const commentInfo = {
+		owner: context.repo.owner,
+		repo: context.repo.repo,
+		issue_number: context.issue.number,
+	};
+
+	const comments = ( await github.rest.issues.listComments( commentInfo ) )
+		.data;
+
+	for ( const currentComment of comments ) {
+		if (
+			currentComment.user.type === 'Bot' &&
+			currentComment.body.includes( 'Test using WordPress Playground' )
+		) {
+			return;
```

Comment:
Should we be editing a current comment if the build was updated?

Reply:
@jonathansadowski good point, originally the plugin artifact name was stable so there wasn't a need, but now we generate it based on run id, I'll fix this.

Reply:
done here https://github.com/woocommerce/woocommerce/pull/46761/commits/cdd39ff81f5e00c8d9deb9780e23d6a27858369a